README.md

```diff
@@ -1,2 +1,46 @@
 # continuation-local-storage-jvm
-A common JVM implementation of continuation-local storage
+
+This is a common implementation of continuation-local storage for the JVM.
+It is based on Twitter's com.twitter.util.Local, but this is a Java port
```

Comment:
supernit, backtick type names? `com.twitter.util.Local` or even better link to it.